llama.cpp

```diff
@@ -10194,6 +10199,7 @@ static void llama_convert_tensor_internal(
     size_t in_buff_offs = 0;
     size_t out_buff_offs = 0;
 
+    workers.reserve(nthread);
```

Comment:
There is already a reserve() of workers right when it is defined in the caller - reserving it here is pointless.